project_scrum/project_scrum.py

```diff
@@ -0,0 +1,500 @@
+    actor_ids = fields.Many2many(comodel_name='project.scrum.actors', string='Actor')
+    sprint_id = fields.Many2one(comodel_name='project.scrum.sprint', string='Sprint', store=True)
+    us_id = fields.Many2one(comodel_name='project.scrum.us', string='User Stories')
+    date_start = fields.Datetime(string='Starting Date', required=False, default=fields.Datetime.now())
+    date_end = fields.Datetime(string='Ending Date', required=False)
+    use_scrum = fields.Boolean(related='project_id.use_scrum')
+    description = fields.Html('Description')
+    current_sprint = fields.Boolean(compute='_current_sprint', string='Current Sprint', search='_search_current_sprint')
+    moscow = fields.Selection([
+            (1, 'must'), 
+            (2, 'should'), 
+            (3, 'could'), 
+            (4, 'wont'), 
+            ('not_set', 'Not Set'), 
+            ], string='Moscow')
+    value = fields.Selection([
+             (0, '0'),
+             (0.5, '0.5'),
+             (1, '1'),
+             (2, '2'),
+             (3, '3'), 
+             (5, '5'), 
+             (8, '8'), 
+             (13, '13'), 
+             (20, '20'),
+             (40, '40'),
+             (100, '100'),
+             (00, 'Not Set'),
+             ], string='Value')
+    risk = fields.Selection([
+            (0, '0'),
+            (1, '1'),
+            (2, '2'),
+            (3, '3'),
+            (5, '5'),
+            (8, '8'),
+            (13, '13'),
+            (20, '20'), 
+            (40, '40'),
+            (100, '100'),
+            (00, 'Not Set'),
+            ], string='Risk')
+    kano = fields.Selection([
+            ('excitement', 'Excitement'),
+            ('indifferent', 'Indifferent'),
+            ('mandatory', 'Mandatory'), 
+            ('performance', 'Performance'),
+            ('questionable', 'Questionable'),
+            ('reverse', 'Reverse'), 
+             ('not_set', 'Not Set'),
+            ], string='Kano')
+    color = fields.Integer(related='project_id.color')
+    reference = fields.Char('Number', select=True, readonly=True, copy=False, default=':')
+
+    @api.depends('sprint_id')
+    @api.one
+    def _current_sprint(self):
+        sprint = self.env['project.scrum.sprint'].get_current_sprint(self.project_id.id)
+        #~ _logger.error('Task computed %r' % self)
+        if sprint:
+            self.current_sprint = sprint.id == self.sprint_id.id
+        else:
+            self.current_sprint = False
+            
+    def _search_current_sprint(self, operator, value):
+        #~ raise Warning('operator %s value %s' % (operator, value))
+        project_id = self.env.context.get('default_project_id', None)
+        sprint = self.env['project.scrum.sprint'].get_current_sprint(project_id)
+        #~ raise Warning('sprint %s csprint %s context %s' % (self.sprint_id, sprint, self.env.context))
+        _logger.error('Task %r' % self)
+        return [('sprint_id', '=', sprint and sprint.id or None)]
```

Comment:
if you followed up on my above comment, you don't need the `or None` any more, empty browse records yield their field's default value, which is None here